package.json

```
@@ -39,4 +39,8 @@
     "fs": "0.0.1-security",
     "path": "^0.12.7"
   }
+  "publishConfig": { "registry": "https://npm.pkg.github.com/" } ,
+  "repository": {
+    "url": "https://github.com/nhcarrigan/TSProGen"
```

```suggestion
  },
  "publishConfig": { "registry": "https://npm.pkg.github.com/" },
  "repository": {
    "url": "https://github.com/nhcarrigan/TSProGen"
```
Missing comma, extra space. Should fix the linting issues.